bundles/org.openhab.binding.ferroamp/src/main/resources/OH-INF/thing/Ferroamp.xml

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="ferroamp"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<thing-type id="energyhub">
+		<label>ferroamp Energy Hub</label>
+		<description>Bindings for the ferroamp Energy Hub</description>
+		<category>EnergyHub</category>
+		<channels>
+			<channel typeId="ehub-wloadconsq-l1" id="ehubWLoadConsqL1"/>
```

Comment:
Can you adjust all channel id's to lower-case-hyphen
https://www.openhab.org/docs/developer/guidelines.html#naming-convention

Reply:
Please, see commit 20240607

Reply:
Please, see commit 20240607